monai/bundle/scripts.py

```diff
@@ -281,19 +281,22 @@ def load(
 
     if device is None:
         device = "cuda:0" if is_available() else "cpu"
-    # loading with `torch.jit.load`
+    # loading with `torch.jit. load`
```

Comment:
May I know why added a blank here? Is it a typo?

Thanks.

Reply:
Thanks Nic, it seems the code format tool mis-modify this place, I just pushed a new commit.